pom.xml

```diff
@@ -82,6 +82,14 @@
 
         <!-- CVE-2021-44228 Log4Shell, CVE-2021-4104, CVE-2021-45105, CVE-2021-44832 -->
         <log4j.version>2.17.1</log4j.version>
+
+        <!-- Sonar settings -->
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+        <sonar.organization>bf2fc6cc711aee1a0c2a</sonar.organization>
+        <sonar.coverage.jacoco.xmlReportPaths>
+            ${project.basedir}/target/site/jacoco-aggregate/jacoco.xml,
+            ${project.basedir}/../target/site/jacoco-aggregate/jacoco.xml
+        </sonar.coverage.jacoco.xmlReportPaths>
```

Comment:
Not needed since it's passed on the command line (also should have been removed from admin API).